tools/util/ReplayInterpreter.cpp

```diff
@@ -123,7 +123,7 @@ int ReplayInterpreter::getSymbolicOperation( std::string& raw_line, std::string&
     }
     else {
       std::cerr << "Unknown Replay (" << m_json["event"] << ")\n";
```

Comment:
Could this be an UMPIRE_ERROR, or is cerr better?

Reply:
I agree with @jdahm - I think an UMPIRE_ERROR is appropriate here.

Reply:
I'm not sure I agree :).  I would rather have replay limit itself to Umpire interactivity as much as possible and only "replay" the allocations/deallocations.  I would rather not utilize other parts of Umpire if possible.

Reply:
Fair enough - please wrap this up in a `REPLAY_ERROR` macro to avoid duplicating the std::cerr, exit pattern.

Reply:
See my comments on this above.  I think this needs additional discussion.

Reply:
You said you didn't want to use the UMPIRE macro, which is fine. I think this should be wrapped in a replay specific macro to avoid errors due to repetition.

Reply:
Done.  Please mark resolved.